packages/graphql-testing/src/graphql-controller.ts

```diff
@@ -1,20 +1,19 @@
-import {ApolloLink, GraphQLRequest} from 'apollo-link';
 import {
-  ApolloReducerConfig,
+  ApolloClient,
+  ApolloLink,
+  GraphQLRequest,
   InMemoryCache,
-  IntrospectionFragmentMatcher,
-} from 'apollo-cache-inmemory';
-import {ApolloClient} from 'apollo-client';
+  InMemoryCacheConfig,
+} from '@apollo/client';
 
-import {TestingApolloClient} from './client';
 import {MockLink, InflightLink} from './links';
 import {Operations} from './operations';
 import {operationNameFromFindOptions} from './utilities';
 import {GraphQLMock, MockRequest, FindOptions} from './types';
 
 export interface Options {
-  unionOrIntersectionTypes?: any[];
-  cacheOptions?: ApolloReducerConfig;
+  possibleTypes?: InMemoryCacheConfig['possibleTypes'];
```

Comment:
Should we get rid of the possibleTypes option here, and just have cacheOptions, and down below do `const cache = new InMemoryCache(cacheOptions);`.

Then consumers would do createGraphQLFactory({cacheOptions: {possibleTypes: BLAH}})`